HW_1/Task3.py

```diff
@@ -0,0 +1,27 @@
+# Вы пользуетесь общественным транспортом? Вероятно, вы расплачивались за проезд и получали билет 
+# с номером. Счастливым билетом называют такой билет с шестизначным номером, где сумма первых 
+# трех цифр равна сумме последних трех. Т.е. билет с номером 385916 – счастливый, т.к. 3+8+5=9+1+6. 
+# Вам требуется написать программу, которая проверяет счастливость билета.
+# *Пример:*
+# 385916 -> yes
+# 123456 -> no
+
+tic = int(input('Введите шестизначный номер билета: '))
+n1 = tic // 1000
+n2 = tic % 1000
+sum1 = 0
+sum2 = 0
+
+if tic // 1000000 == 0 and tic // 10000 > 0:
+    while n1 > 0:
+        sum1 += n1 % 10
+        n1 //=10
+    while n2 > 0:
+        sum2 += n2 % 10
+        n2 //=10
```

Comment:
Можно обойтись одним циклом
```suggestion
    while n1 > 0 or n2 > 0:
        sum1 += n1 % 10
        n1 //= 10
        sum2 += n2 % 10
        n2 //= 10
```